### ISSUE TITLE
Rectangular box problem about S3DIS dataset visualization?

### ISSUE THREAD

**Issue by MichaleWong** [rusty1s/pytorch_geometric#1623]
## ❓ Questions & Help
it's the area6 of s3dis dataset, why it lools like a rectangular box?
I check the range of x and y , it is between-0.5 and 0.5, but z is from 0.0 to 4.273, does any one know what's wrong with t that, by the way, I used open3d to visualize it
![2020-09-09 16-43-19屏幕截图](https://user-images.githubusercontent.com/6574543/92577989-dbe24000-f2bd-11ea-9ffd-752e379d38f6.png)

```
def writeO3D(dataset, file_path):
    points = dataset.data.pos.cpu().numpy()
    colors = dataset.data.x[:,:3].cpu().numpy()
    ply = o3d.geometry.PointCloud()
    ply.points = o3d.utility.Vector3dVector(points)
    ply.colors = o3d.utility.Vector3dVector(colors)
    o3d.io.write_point_cloud(file_path, ply)


test_dataset  = S3DIS(path,test_area=6, train=False)
gactest.writeO3D(test_dataset,"/home/michael/Data/S3DIS/area6.ply")
```

**Comment by tchaton** [comment]
Hey @MichaleWong,

Pytorch Geometric s3dis is the 1x1 square cylinder S3DIS dataset.
You will find 3 versions there: https://github.com/nicolas-chaulet/torch-points3d

Best,
Thomas Chaton

**Comment by banciuadrian** [comment]
Hey @tchaton 

Sorry if asking this here is improper, but this may help people with visualization as well later on and it seems proper to ask here.

Loading the data with S3DIS1x1 compared to the OriginalFused results in a different number of points. I am using Area 5 only.
With Fused I get 78649818 points and with S3DIS1x1 I get about 4096*6852 points, which is about 50 million less. I want to visualize the predicted semantic segmentation labels of my model, but I can't make a one to one correspondence due to the Fused version having so many more points.

In this case, how do I go about visualizing the prediction from my model on the whole area that has been trained using the PyTorch Geometric S3DIS dataset? (ideally I want to load the fused area 5 fully and see the predictions on that, i.e., reduce it to the 28 million dimension and have the positions be spatially accurate, which is what I was trying to do by using the Fused "loader")

Best regards, 
Adrian
